infrastructure/router/router.go

```diff
@@ -0,0 +1,31 @@
+package router
+
+import (
+	"WizelineApi/interface/controller"
+	//"academy-go-q42021/interface/controller"
+	"log"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+)
```

Comment:
Best practices suggest we need to have our imports separated into three blocks.
1. Standard libraries
2. Local packages
3. 3rd party packages

```suggestion
import (
	"fmt"
	"log"
	"net/http"
	
	"WizelineApi/interface/controller"
	
	"github.com/gorilla/mux"
)
```

Please fix this in all other occurrences.